### ISSUE TITLE
Classification error

### ISSUE THREAD

**Issue by AjinkyaBankar** [aniketdali/vggnet#1]
Hello Sir,
The following path has the weights in .txt format:
https://mega.nz/#!LIhjXRhQ!scgNodAkfwWIUZdTcRfmKNHjtUfUb2KiIvfvXdIe-vc

Or do we need to convert them using keras_weights_converter.py script again?
When I make the executable, the application named 'base' is created. I execute it using the weights.txt file downloaded from the above link. It is not asking for the input image.txt file. And directly giving the following output in the softmax.txt file:
accuracy:0.001980 class:0 
If I specify the input image as vol.txt, which you provided, then also it gives:
 accuracy:0.001980 class:0 
Kindly help to get the correct results. Thank you.
